### ISSUE TITLE
Save to same folder as input files

### ISSUE THREAD

**Issue by eduperez** [jcelaya/hdrmerge#97]
When saving, hdrmerge always remembers the last folder where a file was saved; I think it would be more useful to change to the folder where the input files are located

**Comment by heckflosse** [comment]
@eduperez That would be useful for one workflow, but not for other workflows.
Wouldn't it be better to remember the destination folder in the way RawTherapee does in queue processing?

![savemerged](https://cloud.githubusercontent.com/assets/13733487/19833057/50042412-9e35-11e6-91b1-7151d90ba815.png)

**Comment by Beep6581** [comment]
I agree with @heckflosse that what's good for one workflow might not be good for another, but I also agree with @eduperez that saving to the source folder would be the more common scenario and so should be the default behavior.

RawTherapee adds a temporary shortcut ("bookmark") in the load/save file windows, the shortcut points to the source folder, but this behavior isn't implemented for all file panels (I think it's only in the "Save image" and "Load/Save PP3" windows) and users often don't find the shortcut unless we point it out in the forum.
